lib/constants.js

```
@@ -0,0 +1,4 @@
+const BASE_URL = "https://cravyn-web-frontend.vercel.app",
+  NAME = "Cravyn - Crave, Click, Delivered";
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)